app/(dashboard)/[category]/conversation/nextTicketPreview.tsx

```
@@ -0,0 +1,117 @@
+    : (mailboxPreferences?.preferences?.showNextTicketPreview ?? true); // Use actual preference once loaded
+
+  // Don't render on server or if not mounted or if explicitly disabled
+  if (!mounted || !conversationListData?.conversations || showNextTicketPreview === false) return null;
+
+  const conversations = conversationListData.conversations;
+  const nextConversation = conversations[currentIndex + 1] || conversations[0];
+
+  // Don't show preview if there's only one conversation or no next conversation
+  if (conversations.length <= 1 || !nextConversation) return null;
+
+  const isLastConversation = currentIndex === conversations.length - 1;
```

ESLint

[error] 'isLastConversation' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)